src/Molecules/ProfileCard/ProfileCard.js

```diff
@@ -1,14 +1,19 @@
+import propTypes from 'prop-types';
+
 import ProfileCardStyled from './ProfileCardStyled';
-import UserAddress from '../../Atoms/UserAdress/UserAddress';
-import UserNameAndImg from '../../Atoms/UserNameAndImg/UserNameAndImg';
+import UserProfileData from '../../Atoms/UserProfileData/UserProfileData';
 
-const ProfileCard = () => {
+const ProfileCard = ({ userData }) => {
   return (
     <ProfileCardStyled>
-      <UserNameAndImg />
-      <UserAddress />
+      <div className="bg-box" />
+      <UserProfileData userData={userData} />
```

Comment:
If this is context data, do not pass it down, use context in the component that needs it

Reply:
Thanx for this comment. Refactored